contents/blog/alerts-examples.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Never miss changes to your key metrics: 5 alerts you can use'
+date: 2025-01-07
+author:
+ - bijan-boustani
+rootpage: /blog
+category: General
+tags:
+  - Alerts
+  - Product updates
+---
+
+To keep life interesting, it's important to live in a state of perpetual ignorance. When things are going well, there's no need to find out why. When things go downhill, let anxiety run its course. After all, who needs data and insights when you have hope?
+
+![Driving blind](https://res.cloudinary.com/dmukukwp6/image/upload/319051_c8e072f85e.gif)
```

Comment:
Does putting this in a `<ProductScreenshot>` center it?

Reply:
I tried this out as a `ProductScreenshot` but it doesn't center the image and looked a little weird with the extra border. I ended up reverting the change since I think most of our meme images use Markdown images.